easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2015a.eb

```diff
@@ -17,6 +17,6 @@ source_urls = [
 
 patches = ['METIS_IDXTYPEWIDTH.patch']
 
-builddependencies = [('CMake', '3.2.1')]
+builddependencies = [('CMake', '3.0.0')]
```

Comment:
please don't change dependency versions (let alone *downgrade* them), you may introduce conflicts higher up